opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -99,10 +99,11 @@ message Span {
   // the same display name at the same call point in an application.
   // This makes it easier to correlate spans in different traces.
   //
-  // This field is semantically required to be set to non-empty string.
-  // When null or empty string received - receiver may use string "name"
-  // as a replacement. There might be smarted algorithms implemented by
-  // receiver to fix the empty span name.
```

Comment:
Why is this outdated? There may be source that do not follow the semantics, in that case the receiver does either reject the span or treats that with a "default"

Reply:
Such behavior (replacing empty span name with default or rejecting the span) is not mentioned anywhere in the specification. Also, it does not seem to be implemented anywhere

Reply:
That is not true since in OpenCensus we had a TrcuncatableString which was a message so was able to be null

Reply:
This was added to support languages like Java where strings can be null but in prptobufs this cannot be possible

Reply:
Redone this, so I believe this is no longer relevant and for clarity I will resolve this conversation